SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialStatics.cpp

```diff
@@ -72,7 +73,13 @@ bool USpatialStatics::IsSpatialOffloadingEnabled(const UWorld* World)
 	{
 		if (const ASpatialWorldSettings* WorldSettings = Cast<ASpatialWorldSettings>(World->GetWorldSettings()))
 		{
-			return IsSpatialNetworkingEnabled() && WorldSettings->WorkerLayers.Num() > 0 && WorldSettings->IsMultiWorkerEnabled();
+			if (!IsSpatialNetworkingEnabled() || !WorldSettings->IsMultiWorkerEnabled())
+			{
+				return false;
+			}
+
+			const UAbstractSpatialMultiWorkerSettings* MultiWorkerSettings = WorldSettings->MultiWorkerSettingsClass->GetDefaultObject<UAbstractSpatialMultiWorkerSettings>();
+			return MultiWorkerSettings->WorkerLayers.Num() > 1;
```

Comment:
We used to check `WorldSettings->WorkerLayers.Num() > 0` which seems incorrect. Or was that because the default layer wasn't added previously and is now?

Reply:
the latter